tests/unit/functions.tcl

```diff
@@ -288,6 +288,28 @@ start_server {tags {"scripting"}} {
         r function flush async
         assert_match {} [r function list]
 
+        # LAZYFREE_THRESHOLD is 64
+        for {set i 0} {$i < 100} {incr i} {
+            r function load REPLACE [get_function_code lua test$i test$i {local a = 1 while true do a = a + 1 end}]
+        }
+        assert_morethan [s used_memory_vm_functions] 70000
+        r config resetstat
+        r function flush async
+        # Wait for the completion of lazy free for both functions and engines.
+        set start_time [clock seconds]
+        while {1} {
+            # Tests for race conditions between async function flushes and main thread Lua VM operations.
+            r function load REPLACE [get_function_code lua test test {local a = 1 while true do a = a + 1 end}]
```

Comment:
do we need the REPLACE arg?

Reply:
yes, if not we can't load the same function, here i just want to keep touching the lua vm.
btw, thread ASAN reported the race issue in the original code with this test.